package.json

```diff
@@ -1,66 +1,66 @@
-{
-  "name": "ccf-ui",
-  "version": "1.0.0-alpha.1",
-  "scripts": {
-    "ng": "ng",
-    "start": "ng serve",
-    "build": "ng build --prod",
-    "build:demo": "ng build --prod --output-hashing=none --base-href=./",
-    "build:doc": "npm run compodoc",
-    "build:netlify": "npm-run-all build cp-netlify-redirects compodoc",
-    "cp-netlify-redirects": "cp src/_redirects dist/ccf-ui/_redirects",
-    "compodoc": "npx compodoc -p src/tsconfig.app.json --output dist/ccf-ui/docs",
-    "test": "ng test",
-    "test:wsl": "ng test --browsers=ChromeHeadlessWSL",
-    "lint": "ng lint",
-    "e2e": "ng e2e",
-    "sonar": "sonar-scanner"
-  },
-  "private": true,
-  "dependencies": {
-    "@angular/animations": "~9.1.2",
-    "@angular/cdk": "^9.2.1",
-    "@angular/common": "~9.1.2",
-    "@angular/compiler": "~9.1.2",
-    "@angular/core": "~9.1.2",
-    "@angular/forms": "~9.1.2",
-    "@angular/material": "^9.2.1",
-    "@angular/platform-browser": "~9.1.2",
-    "@angular/platform-browser-dynamic": "~9.1.2",
-    "@angular/router": "~9.1.2",
-    "@ngxs-labs/data": "^2.4.1",
-    "@ngxs/store": "^3.6.2",
-    "lodash": "^4.17.15",
-    "ng5-slider": "^1.2.4",
-    "rxjs": "~6.5.5",
-    "tslib": "^1.11.1",
-    "zone.js": "~0.10.3"
-  },
-  "devDependencies": {
-    "@angular-devkit/build-angular": "~0.901.1",
-    "@angular/cli": "~9.1.1",
-    "@angular/compiler-cli": "~9.1.2",
-    "@angular/language-service": "~9.1.2",
-    "@compodoc/compodoc": "^1.1.11",
-    "@types/jasmine": "^3.5.10",
-    "@types/jasminewd2": "~2.0.3",
-    "@types/js-yaml": "^3.12.3",
-    "@types/lodash": "^4.14.150",
-    "@types/node": "^13.13.1",
-    "codelyzer": "^5.2.2",
-    "jasmine-core": "~3.5.0",
-    "jasmine-spec-reporter": "^5.0.1",
-    "karma": "^5.0.2",
-    "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage-istanbul-reporter": "~2.1.0",
-    "karma-jasmine": "^3.1.1",
-    "karma-jasmine-html-reporter": "^1.5.3",
-    "npm-run-all": "^4.1.5",
-    "protractor": "^5.4.4",
-    "shallow-render": "^9.0.4",
-    "sonar-scanner": "^3.1.0",
-    "ts-node": "~8.3.0",
-    "tslint": "^6.1.1",
-    "typescript": "~3.7.5"
-  }
-}
+{

```

Comment:
Why is this showing as committed? I don't see any text changes. Formatting?

Reply:
Not sure what happened there. It should all be the same...

Reply:
Reverted the commit